openlabcmd/openlabcmd/plugins/base.py

```diff
@@ -45,16 +37,19 @@ class Plugin(object):
     ptype = 'Base'
     name = 'Base Check'
     failed = False
-    reasons = []
     nocolor = False
     experimental = False
 
+    def __init__(self):
+        self.reasons = []
+
     def register_signals(self):
         # print("%s has been loaded." % self.__class__.__name__)
         pass
 
-    def check_begin(self):
-        pass
+    @classmethod
```

Comment:
-1, As I mentioned before, it's better to use ``__init__`` to create new plugin instance.